src/executor/query/UnwindExecutor.cpp

```diff
@@ -17,27 +17,27 @@ folly::Future<Status> UnwindExecutor::execute() {
     SCOPED_TIMER(&execTime_);
 
     auto *unwind = asNode<Unwind>(node());
-    auto columns = unwind->columns()->columns();
-    DCHECK_GT(columns.size(), 0);
-
-    auto iter = ectx_->getResult(unwind->inputVar()).iter();
-    DCHECK(!!iter);
+    auto &inputRes = ectx_->getResult(unwind->inputVar());
```

Comment:
What's improvement?

Reply:
Removed a inner for-block to avoid expression overhead.